examples/hybrid_jobs/1_Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs/Quantum_machine_learning_in_Amazon_Braket_Hybrid_Jobs.ipynb

```diff
@@ -60,25 +60,26 @@
      "name": "stdout",
```

Comment:
<p>Is it worth showcasing the use of the decorator as a standalone function? More specifically, does this work?</p><pre class="ql-syntax" spellcheck="false">train_circuit_hybrid_job = hybrid_job(
    train_circuit,
    device=None,
    local=False,
    include_modules="qcbm",
    input_data="data.npy"
)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
In general that should work. That would be equivalent to decorating `train_circuit` and could be invoked with `train_circuit_hybrid_job(...)`, though I don't know if it's worth showcasing this specifically. At that point, might as well just use the create pattern. I think it will be apparent for anyone familiar enough with python to want to do this for a specific reason